test/envoye2e/stackdriver_plugin/fake_stackdriver/timeseries.go

```diff
@@ -63,7 +63,7 @@ const ClientRequestCountJSON = `{
 	   "type":"istio.io/service/client/request_count",
 	   "labels":{
 		  "destination_owner":"kubernetes://api/apps/v1/namespaces/default/deployment/ratings-v1",
-		  "destination_port":"20019",
+		  "destination_port":"20017",
```

Comment:
This seems closer to the truth. Previously, upstream remote port was reported as destination port.

Reply:
Value 0 requires further investigation.